api/tests/app/billing/test_billing.py

```diff
@@ -34,6 +34,7 @@ def _assert_dict_equals(actual, expected_dict):
     assert actual == expected_dict
 
 
+@pytest.mark.skip(reason='database reset issues')
```

Comment:
Are we able to elaborate or record what we know about the issue? Will help future us (and current me) know we're skipping the tests?

Reply:
I might have a stab at fixing these tests as I think they're the ones I modified.